### ISSUE TITLE
Not an issue, a question on understanding 

### ISSUE THREAD

**Issue by GabriellaS-K** [rstudio/bookdown#1043]
Please forgive the v basic question. I have recently started using R bookdown, to try write scientific papers. I've read the cookbook and book down guide and looked at examples too. My question isn't about an issue with the code, its that I don't understand' how it works. I've been struggling with this for 5 days now, and thought I should ask for help.  I use rmardown, and understand that, but I don't understand how bookdown 'knows' what to put together. As instructed by the helpsheets, I have several chapters:

an index
01-introduction (mainly free text)
02 methods (mainly tables)
03 results (outputs, graphs, tables)
04 concolusion (free text).
Normally in rmarkdown, every new file I open, I have a 'r set up' chunk where I set the initial global decisions for the doc (echo=TRUE for example) and have YAML in each doc. I also install and load all packages I need for that document. Do I still need to do this in bookdown for each chapter? Or do I do it only once in the index chapter? I just don't quite understand what to put in index and what not to put in the other chapters.

Lastly: If I create a graph in chapter 3 can I put it in chapter 2? Or does it render in order of chapters, and is the better way to have a separate file for all the code for my graphs?

I think the issue is, I don't actually understand the YAML in the examples and what its saying-particularly the output parts (probably also getting confused as I've looked at thesis down too...its all a big mess in my mind now!). 

Thanks so much and sorry for such a basic question. I'm relatively new to all this (only started using markdown a few months ago!
---

By filing an issue to this repo, I promise that

- [X] I have fully read the issue guide at https://yihui.org/issue/.
- [X] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('bookdown')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('rstudio/bookdown')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [X] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by cderv** [comment]
👋 Hi @GabriellaS-K, and welcome in the community of **bookdown** users ! 

Best way to understand how **bookdown** works would I think be to follow the Introduction section in the bookdown's book. There is a [Get Started](https://bookdown.org/yihui/bookdown/get-started.html) part where you will download a demo book to render it, play with and understand how it works. Have you already follow the steps in this chapter ?  The demo book is available online: https://github.com/rstudio/bookdown-demo

The aim of **bookdown** is to be able to write a "book-like" content using several Rmd files, usually one per chapter. You can find a lot of example online - search in https://bookdown.org/ and you could find the source code of each book too usually. 

Usually you will have an `index.Rmd` which will be the first document of your book where you define some configurations in the YAML header, and then you will have some other Rmd (one per chapter) without any YAML. It is mentioned in the [Usage](https://bookdown.org/yihui/bookdown/usage.html) section. 

Regarding how **bookdown** renders, you'll find some information in the chapter regarding [rendering approach](https://bookdown.org/yihui/bookdown/new-session.html). With the default Merge and Knit approach, you will be able to consider your whole book composed of several Rmd files as one Rmd files. That means you could access results from a chapter in another, usually after it has been created. But you can use advanced tricks to use a result before (in Chapter 2 while defined in Chapter 3) - it involved a cache-like approach and two rendering: you have an example in the [cookbook](https://bookdown.org/yihui/rmarkdown-cookbook/load-cache.html)

Basically, with the example you shown you'll have 
* Your YAML defined in index.Rmd with the first page of your book. (see bookdown-demo [source](https://github.com/rstudio/bookdown-demo/blob/master/index.Rmd#L2) and [output](https://bookdown.org/yihui/bookdown-demo/))
* Your content in the other Rmd doc without any YAML header - see also bookdown-demo
* With the default approach, any configuration you set in `index.Rmd` will apply in each following Rmd rendering - so you can set your knitr option in the `index.Rmd`

At last, what did not work for you in your example ? What difficulties did you encounter during the 5 days ? 

I am happy to help if you can share more. Know you can also ask questions on https://community.rstudio.com where the is a broad community ready to help !

**Comment by GabriellaS-K** [comment]
Hi!! 

This was all incredibly helpful! I have used your links, done some more reading, and it all makes much more sense now, errors are all fixed! 

Thank you for your time
